frontend/src/containers/AdminPanel/AdminPanel.js

```diff
@@ -36,6 +39,14 @@ class AdminPanel extends Component {
         users: this.createFullName(data),
       });
     });
+
+    const { user } = this.props;
+
+    if (isNil(user)) {
+      setTimeout(() => {
```

Comment:
This feels kinda hacky, but I'm unsure of a better way to fix it. I've tried some things locally to no avail, so I guess it's fine for now.

@vhoof we may want to backlog a ticket for making this less hacky at some point.

Reply:
Yeah, I'm sorting it out in a different branch.